docs/user/tutorials/01-50-expose-and-secure-a-workload/01-50-expose-and-secure-workload-oauth2.md

```diff
@@ -49,43 +49,52 @@ This tutorial shows how to expose and secure Services using APIGateway Controlle
    ```
 
 2. Get tokens to interact with secured resources using the client credentials flow:
-   <div tabs>
-     <details>
-     <summary>
-     Token with `read` scope
-     </summary>
-     * Export the following value as an environment variable:
-        ```shell
-        export KYMA_DOMAIN={KYMA_DOMAIN_NAME}
-        ```  
-     * Get the opaque token:
-        ```shell
-        curl --location --request POST "$TOKEN_URL?grant_type=client_credentials" --header "Content-Type: application/x-www-form-urlencoded" --header "Authorization: Basic $ENCODED_CREDENTIALS"
-        ```
-     * Export the issued token as an environment variable:
-        ```shell
-        export ACCESS_TOKEN_READ={ISSUED_READ_TOKEN}
-        ```
-     </details>
-     <details>
-     <summary>
-     Token with `write` scope
-     </summary>
-     * Export the following value as an environment variable:
-        ```shell
-        export KYMA_DOMAIN={KYMA_DOMAIN_NAME}
-        ```  
-     * Get the opaque token:
-        ```shell
-        curl --location --request POST "$TOKEN_URL?grant_type=client_credentials" --header "Content-Type: application/x-www-form-urlencoded" --header "Authorization: Basic $ENCODED_CREDENTIALS"
-        ```
-     * Export the issued token as an environment variable:
-        ```shell
-        export ACCESS_TOKEN_WRITE={ISSUED_WRITE_TOKEN}
-        ```
-      </details>
-   </div>
 
+    <div tabs name="export-values">
+    <details>
+    <summary>
+    Token with `read` scope
+    </summary>
+      
+    * Export the following value as an environment variable:
```

Comment:
I'm assuming the order of these steps is important, so I'd suggest an ordered list.

```suggestion
    1. Export the following value as an environment variable:
```

If you do that, all indented lines probably need an additional blank space, because numbered lists need 3 instead of 2...